api/StorageManager.json

```diff
@@ -5,34 +5,56 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/StorageManager",
         "support": {
           "webview_android": {
-            "version_added": null
+            "version_added": "48"
           },
           "chrome": {
-            "version_added": null
+            "version_added": "48"
```

Comment:
MDN stated `55`, but basic support was actually already implemented in `48`:
https://chromium.googlesource.com/chromium/src/+/21ba553558ed7cd65f59199b28dd8732f5dc93a0%5E%21/third_party/WebKit/Source/modules/quota/StorageManager.idl